Description: Fix FTBFS with GCC 4.7
 src/scheduler/src/client/Client.cc: In member function 'void Client::read_oneauth(std::string&)':
 src/scheduler/src/client/Client.cc:70:34: error: 'getuid' was not declared in this scope
Author: Damien Raude-Morvan <drazzib@debian.org>
Last-Update: 2012-05-08
Forwarded: no
--- a/src/scheduler/src/client/Client.cc
+++ b/src/scheduler/src/client/Client.cc
@@ -25,6 +25,7 @@
 #include <limits.h>
 #include <string.h>
 #include <stdlib.h>
+#include <unistd.h>
 
 #include <sstream>
 
